base/broadcast.jl

```diff
@@ -826,8 +829,14 @@ preprocess_args(dest, args::Tuple{}) = ()
         end
     end
     bc′ = preprocess(dest, bc)
-    @simd for I in eachindex(bc′)
-        @inbounds dest[I] = bc′[I]
+    if is_simd_safe(dest) && is_simd_safe(bc′)
```

Comment:
`bc` is read-only memory – I don't think it matters whether it is safe to write to it